app/src/main/java/com/sopt/now/domain/model/AuthData.kt

```diff
@@ -0,0 +1,10 @@
+package com.sopt.now.domain.model
+
+import android.provider.ContactsContract.CommonDataKinds.Phone
+
+data class AuthData(
```

Comment:
```suggestion
data class AuthEntity(
```
domain 영역의 data class는 보통 Entity로 네이밍 지어요

각 영역별 네이밍(보통 이렇게 지어요)
1. ui : model
2. domain : entity
3. data : dto